packages/storybook-ui-components/layout/StoryDescription.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FunctionComponent } from "react";
+
+const StoryDescription: FunctionComponent = ({ children }) => (
+  <div
+    style={{
+      margin: "2rem 0",
+      fontSize: "14px",
+      color: "#475872",
+      maxWidth: "600px",
+      lineHeight: "24px",
```

Comment:
This is more of a question but I'm interested in the use of `rem` vs `px` or `%`.  I found [this](https://engageinteractive.co.uk/blog/em-vs-rem-vs-px) on a _very_ cursory search.  

I'm mostly naive to the world of `rem` and `em` but I'm wondering if the included link above seems to you like a decent summary of where we want to be?  If so when and to what degree would be ideal for `rem`-ifying our DS code?

Reply:
I am a fan of `rem`. In my personal work, I try to use them exclusively (except in media queries, as the article points out, I use `em`). I think this article does a good job of hitting the high points. For me the seminal argument for _root ems_ was made by [Brent Jackson](https://jxnblk.com/) in [Mathematical Web Typography](https://medium.com/@jxnblk/mathematical-web-typography-d69186780a41). It all came together for me when I realized that, 

```
1px   | 0.0625rem
2px   | 0.125rem
3px   | 0.1875rem
4px   | 0.25rem
5px   | 0.3125rem
6px   | 0.375rem
7px   | 0.4375rem
8px   | 0.5rem
9px   | 0.5625rem
10px  | 0.625rem
11px  | 0.6875rem
12px  | 0.75rem
13px  | 0.8125rem
14px  | 0.875rem
15px  | 0.9375rem
16px  | 1rem
```

Which is pretty clean compared to some other percentage math that can occur in CSS. Generally if we keep any value in CSS as an even number and a factor of 16, then rems aren't so bad.

```
// factors of 16
1px   | 0.0625rem
2px   | 0.125rem
4px   | 0.25rem
8px   | 0.5rem
16px  | 1rem
```

Anyway, I would love to define a _size and space scale_ for the design system but in the meantime I am pro rendering all font sizes and DOM spacing in `rem`.